examples/bluetooth/nimble/ble_phy/phy_cent/tutorial/Ble_Phy_Central_Example_Walkthrough.md

```diff
@@ -0,0 +1,297 @@
+
+main function calls  `ble_store_config_init()` to configure the host by setting up the storage callbacks which handle the read, write, and deletion of security material.
+```c
+/* XXX Need to have a template for store */
+    ble_store_config_init();
+```
+
+
+The main function ends by creating a task where nimble will run using `nimble_port_freertos_init()`. This enables the nimble stack by using `esp_nimble_enable()`. 
+```c
+nimble_port_freertos_init(blecent_host_task);
+
+```
+`esp_nimble_enable()` create a task where the nimble host will run. It is not strictly necessary to have a separate task for the nimble host, but since something needs to handle the default queue, it is easier to create a separate task.
+
+
+
+## Intializaion of LE PHY to Default 1M PHY .
+ 1M PHY is the default PHY for BLE devices which enables it to provide a data rate of 1 Mbps. It is used while establishing the connection between devices and maintains backward compatibility with all those devices that don't have BLE5.0 support.`set_default_le_phy_before_conn()` function set default LE PHY before establishing a connection.
+
+```c
+void set_default_le_phy_before_conn(uint8_t tx_phys_mask, uint8_t rx_phys_mask)
+ {
+    int rc = ble_gap_set_prefered_default_le_phy(tx_phys_mask, rx_phys_mask);
+     if (rc == 0) {         MODLOG_DFLT(INFO, "Default LE PHY set successfully; tx_phy = %d, rx_phy = %d",
+                     tx_phys_mask, rx_phys_mask);
+     } else {
+         MODLOG_DFLT(ERROR, "Failed to set default LE PHY");
+     }
+ }
+ 
+ ```
+
+## Setting LE PHY to 2M PHY.
+
+2M PHY is introduced in BLE5.0 to increase the symbol rate at the physical layer. It provides a symbol rate of 2 Mega symbols per second where each symbol corresponds to a single bit. This allows the user to double the number of bits sent over the air during a given period, or conversely reduce energy consumption for a given amount of data by having the necessary transmit time. 
+
+The following lines change the default LE PHY to 2M PHY.
+
+` tx_phys_mask = BLE_HCI_LE_PHY_2M_PREF_MASK`
+` rx_phys_mask = BLE_HCI_LE_PHY_2M_PREF_MASK` 
+
+```c
+void set_prefered_le_phy_after_conn(uint16_t conn_handle)
+  {
+      uint8_t tx_phys_mask = 0, rx_phys_mask = 0;
+  
+     tx_phys_mask = BLE_HCI_LE_PHY_2M_PREF_MASK;
+ 
+     rx_phys_mask = BLE_HCI_LE_PHY_2M_PREF_MASK;
+ 
+     int rc = ble_gap_set_prefered_le_phy(conn_handle, tx_phys_mask, rx_phys_mask,0);
+     if (rc == 0) {
+         MODLOG_DFLT(INFO, "Prefered LE PHY set to LE_PHY_2M successfully");
+     } else {
+         MODLOG_DFLT(ERROR, "Failed to set prefered LE_PHY_2M");
+     }
+ }
+ ```
+ 
+   
+   
+
+## Read Operation
+
+`blecent_read` reads the supported LE PHY characteristic. If the peer does not support a required service, characteristic, or descriptor, then the peer lied when it claimed support for the alert notification service. When this happens, or if a GATT procedure fails this function immediately terminates the connection.
```

Comment:
"Then the peer lied when it claimed ...."   . This sounds too harsh in a tutorial.

Suggested wording change:
"If the peer does not support a required service, characteristic, or descriptor OR if a GATT procedure fails , then this function immediately terminates the connection."

Reply:
Thank you @rahult-github  for reviewing the walkthrough. I will do the all mentioned changes and modify this walkthrough